neuralmagic/tests/skip-for-remote-push.txt

```diff
@@ -0,0 +1,41 @@
+tests/metrics/test_metrics.py
+tests/kernels/test_prefix_prefill.py
+tests/kernels/test_pos_encoding.py
+tests/kernels/test_activation.py
+tests/kernels/test_moe.py
+tests/kernels/test_layernorm.py
+tests/kernels/test_attention.py
+tests/distributed/test_custom_all_reduce.py
+tests/distributed/test_comm_ops.py
+tests/prefix_caching/test_prefix_caching.py
+tests/models/test_mistral.py
+tests/models/test_compressed_memory.py
+tests/models/test_marlin.py
+tests/models/test_compressed.py
```

Comment:
I think we should keep this sparsity correctness test around

Reply:
cool.